### ISSUE TITLE
Negated globs not working

### ISSUE THREAD

**Issue by ghost** [lukeed/taskr#106]
Hi there,

If I set up a very basic project:

`mkdir test && touch one.js && touch two.js && touch flyfile.js && npm install fly`

and into the flyfile dump:

``` javascript
export default function* () {
  yield this.source('*.js', '!two.js')
    .target('.tmp')
}
```

Then the result is:

```
.tmp
  one.js
  two.js
  three.js
```

The docs state that `two.js` should be omitted from here.

It would be really nice if fly had the same syntax for globs as the new style in gulp4:

> Note that globs are evaluated in order, which means this is possible:
> 
> `// exclude every JS file that starts with a b except bad.js`
> `gulp.src(['*.js', '!b*.js', 'bad.js'])`

Do you have any plans to support this kind of globbing.

Thanks for all your hard work on fly so far, a really great system

**Comment by ghost** [comment]
Thanks, yes, I think this should work.

We are just wrapping [glob](https://github.com/isaacs/node-glob). The relevant bits should be [this](https://github.com/flyjs/fly/blob/master/src/fly.js#L91) and maybe [this](https://github.com/flyjs/fly-util/blob/master/src/expand.js).

I wonder if this could be a bug perhaps introduced in https://github.com/flyjs/fly/commit/aa756c19c231ee27e3d73dda62772bda0c2cf7fb ?

Thoughts? @andrewsokolov

**Comment by sindresorhus** [comment]
`glob` no longer support negated globs , but rather have an `ignore` option. You could use [`globby`](https://github.com/sindresorhus/globby), which wraps `glob` with a better API, if you want support for negated globs and more.

**Comment by ghost** [comment]
Thanks @sindresorhus Do you have any clue why the maintainer of glob decided to deprecate `!` to create ignore patterns?

**Comment by sindresorhus** [comment]
I don't remember exactly. Something about people using it incorrectly. The `ignore` option was added here: https://github.com/isaacs/node-glob/pull/151

**Comment by ghost** [comment]
Alright, thanks! I'll update it later then and ping you again.

**Comment by lukeed** [comment]
closed in 0.8.7. 

fly-util now uses globby, as per sindre's suggestion :+1:

**Comment by ghost** [comment]
:+1: Great.
